frontend/dispatcher.lua

```diff
@@ -54,43 +54,45 @@ local settingsList = {
     history = {category="none", event="ShowHist", title=_("History"), general=true},
     favorites = {category="none", event="ShowColl", arg="favorites", title=_("Favorites"), general=true},
     filemanager = {category="none", event="Home", title=_("File browser"), general=true, separator=true},
-
+    --
```

Comment:
And in the context of the comment I just left this confuses me greatly. The newline provides the clarifying separation, what need is there for a visual distraction. :-P

Reply:
Dashes because
-sections are divided with empty lines
-look like separators
https://github.com/koreader/koreader/blob/27104ea011da1222e4372ce6118e1829fd1830c9/frontend/ui/elements/filemanager_menu_order.lua#L18-L20